src/Docker.php

```diff
@@ -14,13 +14,25 @@ class Docker
      * @param  string  $path
      * @param  string  $project
      * @param  string  $environment
+     * @param  array  $cliBuildOptions
      * @param  array  $cliBuildArgs
      * @return void
      */
-    public static function build($path, $project, $environment, $cliBuildArgs)
+    public static function build($path, $project, $environment, $cliBuildOptions, $cliBuildArgs)
```

Comment:
Should options be after Args here?

Reply:
The build args are literally passed to the command line like `--build-arg="KEY=VALUE"`, where as the build options are passed in as they are `--KEY=VALUE`. I considered changing those too, though I didn't think it was necessary.

Reply:
@taylorotwell @joedixon